### DISCUSSION TITLE
List protected branches just below default branch when selecting branches in drop-down menu

### DISCUSSION THREAD

**Discussion by scpeters** [community/community#16741] [Other Feature Feedback, Questions, & Ideas]
We have some repositories with lots of branches, and when we make pull requests we are most often targeting a protected branch. If we are lucky, it is the default branch, which means it is easy to find the branch when creating a new pull request, but if it is a protected branch not near the beginning of the alphabet, I will have to scroll or type a few letters of the branch name in order to find it. It would be convenient if the protected branches were shown right below the default branch in the dropdown menu.

<img width="303" alt="Screen Shot 2022-05-16 at 5 40 53 PM" src="https://user-images.githubusercontent.com/3650755/168704316-88521132-ef0a-43a3-aa3c-3d87f0214ffc.png">

**Comment by mathomp4** [comment]
A ➕ 1️⃣  from me. We are the same way. `develop` say is the default branch but `main` is waaaaay down the list sometimes.

**Reply by GiMMC** [reply]
We are the same way. develop say is the default branch but main is waaaaay down the list sometimes.

**Reply by GiMMC** [reply]
We are the same way. develop say is the default branch but main is waaaaay down the list sometimes.
